From ee636d838f4232bad816cba6700b5fbfb9449cfd Mon Sep 17 00:00:00 2001
From: Noam Lessner <nlessner@habana.ai>
Date: Wed, 06 Jul 2022 22:58:03 +0300
Subject: [PATCH] [SW-0] erom: pcie: print char to show loading

Add print to indicate erom_pcie has started, place as soon as possible
to print.

Signed-off-by: Noam Lessner <nlessner@habana.ai>
Change-Id: I5a52204b98586c00c41c2bae480a3f1e3c0131ae
---

diff --git a/app/ext_rom/erom_pcie.c b/app/ext_rom/erom_pcie.c
index 905db81..237a354 100644
--- a/app/ext_rom/erom_pcie.c
+++ b/app/ext_rom/erom_pcie.c
@@ -258,6 +258,7 @@
 	udelay_init();
 
 	hl_uart_init();
+	preboot_puts("\n\reROM\n");
 
 	if (preboot_security_fuse_override()) {
 		preboot_puts("Fake unsecurity fuse override enabled\n");
